### DISCUSSION TITLE
Draw Areas beyond axises limits

### DISCUSSION THREAD

**Discussion by WoodLeg** [airbnb/visx#1864] [Q&A]
Hello everyone, 
I have crawled the web to try and find any leads to achieve this chart:

<img width="640" alt="Screenshot 2024-07-30 at 18 19 49" src="https://github.com/user-attachments/assets/d6efebe2-250d-4a2f-9b30-5f7b401e7e32">

As You can see, the area "bleeds" / go beyond the axises (x-timeseries / y-numbers). I have tried severals libs to achieve this but haven't been able to. My last guess was to try to achieve it directly with D3 which would give me more control over the graph generation.
I understood that `visx` is a collection of primitives rather than a full high-level API like RechartJS. I am using NextJS for the project I am working on and was wondering if:
- generating this chartjs with D3 is possible? Can it be achieved with `visx` as I am using NextJS ?
- if you have any ideas/leads to make this chart design?

I have tried to add data points in the data array that goes over the domain of the scale and it does not work.
I don't have any more ideas on how to achieve this. If you could give me a bit of guidance, it would be much appreciated 🙏 
If you need any more informations, don't hesitate to ask.
Thank you

**Comment by jmssnr** [comment]
Here is a codesandbox that outlines one approach on how to achieve this. Is this helpful to you?
https://codesandbox.io/p/sandbox/4w8zkm
